api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -270,47 +270,25 @@ public static String getFileAsString(File file) throws IOException {
 	 * @param inputStream Stream to copy from
 	 * @param outputStream Stream/location to copy to
 	 * @throws IOException thrown if an error occurs during read/write
+	 * @should not copy the outputstream when outputstream is null
+	 * @should not copy the outputstream if inputstream is null
+	 * @should copy inputstream to outputstream
```

Comment:
Since the test class is a `BaseContextSensitiveTest`  I dont think you can use Mockito to mock the `outputStream` which would make it easy to verify if it has been closed. A solution might be to override the close() of the  `outputStream` method so you can check that it was called see http://stackoverflow.com/a/13800353/3996784

@dkayiwa or @wluyima maybe you can add some more wisdom :smile: